src/Data/List/Membership/DecSetoid.agda

```diff
@@ -30,3 +31,6 @@ x ∈? xs = any? (x ≟_) xs
 
 _∉?_ : Decidable _∉_
 x ∉? xs = ¬? (x ∈? xs)
+
+∈-stable : ∀ {xs ys} → Stable (xs ∈ ys)
```

Comment:
Quantification over `x` and `xs`, surely?
```suggestion
∈-stable : ∀ {x xs} → Stable (x ∈ xs)
```

Reply:
Also, my previous observation about `decidable-stable` was more in the spirit of making the previous versions of the lemma, and in fact *also* this one itself, redundant... (Fairbairn threshhold again)... since `_∈?_` is the one doing the actual heavy lifting?